openassessment/workflow/models.py

```diff
@@ -507,7 +508,10 @@ def cancel(self, assessment_requirements):
             if on_cancel_func is not None:
                 on_cancel_func(self.submission_uuid)
 
-        score = self.get_score(assessment_requirements, step_for_name)
+        try:
+            score = self.get_score(assessment_requirements, step_for_name)
+        except PeerAssessmentRequestError:
```

Comment:
I would suggest a more general error, and add a logging statement in the except.